src/passes/LocalCSE.cpp

```diff
@@ -62,14 +62,17 @@ struct LocalCSE : public WalkerPass<LinearExecutionWalker<LocalCSE>> {
   };
 
   struct UsableHasher {
-    HashType operator()(const Usable value) const {
-      return rehash(uint64_t(value.hashed.hash), value.localType.getID());
+    size_t operator()(const Usable value) const {
+      // Note: Deterministic to standard hash (no longer deterministic)
+      auto digest = wasm::hash(value.hashed.digest);
+      wasm::rehash(digest, value.localType.getID());
+      return digest;
```

Comment:
Afaict this is the only place where a standard hash is constructed from a deterministic hash now, in turn losing deterministic guarantees. The hash is then used as a key of an `unordered_map` in `LocalCSE::checkInvalidations` and `LocalCSE::handle`. Probably irrelevant and just another `unordered_map` in a pass, but thought I mention.